cmd/traefik/traefik.go

```diff
@@ -256,3 +259,41 @@ func checkNewVersion() {
 		}
 	})
 }
+
+func stats(globalConfiguration *configuration.GlobalConfiguration) {
+	if globalConfiguration.SendAnonymousUsage {
+		log.Info(`
+Stats collection is enabled.
+Many thanks to contribute to Traefik improvement by allowing us to receive anonymous information from your configuration.
+Help us improve Traefik by leaving this feature on :)
+More details on: https://docs.traefik.io/basic/#collected-data
+`)
+		collect(globalConfiguration)
+	} else {
+		log.Info(`
+Stats collection is disabled.
+Help us improve Traefik by turning this feature on :)
+More details on: https://docs.traefik.io/basic/#collected-data
+`)
+	}
+}
+
+func collect(globalConfiguration *configuration.GlobalConfiguration) {
+	ticker := time.NewTicker(24 * time.Hour)
+	safe.Go(func() {
+		time.Sleep(10 * time.Minute)
+		err := collector.Collect(globalConfiguration)
+		if err != nil {
+			log.Debug(err)
+		}
+		for {
+			select {
```

Comment:
You don't really need the `select` case here, because `<-ticker.C` will block anyway.